ssh/tailssh/tailssh_integration_test.go

```diff
@@ -122,13 +122,13 @@ func TestIntegrationSSH(t *testing.T) {
 		{
 			cmd:             "pwd",
 			want:            []string{homeDir},
-			skip:            !fallbackToSUAvailable(),
+			skip:            os.Getenv("SKIP_FILE_OPS") != "" || !fallbackToSUAvailable(),
```

Comment:
SKIP_FILE_OPS=0 would still trigger this.

envknob.Bool, or check for "1" explicitly?

Oh, you use "true" below.

Reply:
Done.